guides/common/modules/ref_expiring-rhel-client-support-in-rhel-lifecycle-status.adoc

```diff
@@ -0,0 +1,20 @@
+[id="expiring_rhel_client_support_in_rhel_lifecycle_status_{context}"]
+= Expiring {EL} Client Support in RHEL Lifecycle Status
+
+{Project} provides multiple mechanisms to display information about upcoming End of Support (EOS) events for your {EL} clients.
+
+When either the end of maintenance support or the end of extended lifecycle support approaches in a year, you will see a notification banner in the {ProjectWebUI} if you have hosts in that {EL} version.
+The notification message provides information about the {EL} version, the number of hosts running that version in your environment, the lifecycle support, and the expiration date.
+Along with other information, the {EL} lifecycle column is visible in the report.
+For any hosts that are not running {EL}, *Unknown* will appear in the *Lifecycle Status* and *Report* column.
+
+. In the {Project} web UI, navigate to *Hosts* > *All Hosts*.
+. Click *Manage columns*.
+. Select the  *Content* column to expand.
+. Select *RHEL Lifecycle status*.
+. Click *Save* to generate a new column that displays the {EL} lifecycle status with an icon.
+
+You can also use the *Search* field to search hosts by `rhel_lifecycle_status`, `full_support`, `maintenance_support`, and `approaching_end_of_maintenance`.
```

Comment:
```suggestion
You can also use the *Search* field to search hosts by `rhel_lifecycle_status`.
It can have the following values:
* `full_support`
* ...
```
These are the values I have on my testing instance, you can list them all:
![image](https://github.com/theforeman/foreman-documentation/assets/108661422/a6ccfe26-1f6a-4617-8d03-b2f2216f0893)